azure/eventhub_log_forwarder/activity_log_diagnostic_settings.json

```diff
@@ -0,0 +1,81 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "settingName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the diagnostic setting"
+            }
+        },
+        "resourceGroup": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the relevant Resource Group"
+            }
+        },
+        "eventHubNamespace": {
+            "type": "string",
+            "defaultValue": "datadog-eventhub-namespace",
+            "metadata": {
+                "description": "Name of EventHub namespace"
+            }
+        },
+        "eventHubName": {
+            "type": "string",
+            "metadata": {
+                "description": "Optional. Name of the event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category."
+            }
+        }
+    },
+    "variables": {
+        "subscriptionId": "[subscription().subscriptionId]",
+        "eventHubAuthorizationRuleId": "[concat('/subscriptions/', variables('subscriptionId'), '/resourceGroups/', parameters('resourceGroup'), '/providers/Microsoft.EventHub/namespaces/', parameters('eventhubNamespace'), '/authorizationRules/RootManageSharedAccessKey')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Insights/diagnosticSettings",
+            "apiVersion": "2017-05-01-preview",
```

Comment:
Is there a more updated version for this? Idk if we'd want to use a preview version

Reply:
I believe that is actually the latest according to the docs here: https://docs.microsoft.com/en-us/azure/templates/microsoft.insights/diagnosticsettings

If you click "Latest" it still is that preview version in the example.